pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -532,18 +545,81 @@ func createNodeSecurityGroup(client *gophercloud.ServiceClient, nodeSecurityGrou
 	return nil
 }
 
+//getStringFromServiceAnnotation searches a given v1.Service for a specific annotationKey and either returns the annotation's value or a specified defaultSetting
+func getStringFromServiceAnnotation(service *v1.Service, annotationKey string, defaultSetting string, enabledFlag bool) string {
+	glog.V(4).Infof("getStringFromServiceAnnotation(%v, %v, %v, %v)", service, annotationKey, defaultSetting, enabledFlag)
+	if !enabledFlag {
+		return defaultSetting
+	}
+	if annotationValue, ok := service.Annotations[annotationKey]; ok {
+		//if there is an annotation for this setting, set the "setting" var to it
```

Comment:
This comment seems incorrect - there is no `setting` var

Reply:
+1